examples/1.text_classifier/text_classifier.yaml

```diff
@@ -1,9 +1,9 @@
 name: text-classifier
 
-tokenizer:
-    text_cleaning:
-        rules:
-            - strip_spaces
+#tokenizer:
+#    text_cleaning:
```

Comment:
@frascuchon @dcfidalgo this is not working with 1.0, seems like an issue with nested registrable objects. Besides this is the only place where we use annotations to register components.